src/lib/wallet.ts

```diff
@@ -3,15 +3,16 @@ import * as Types from '../types/wallet';
 import { generateMnemonic, getKeyFromMnemonic } from 'arweave-mnemonic-keys';
 
 const initArweave = (params: Types.InitArweaveProps) => {
-  let arweave;
+  let arweave: Arweave;
```

Comment:
The Arweave transaction functions work as expected in the node environment. Unable to test in web due to some `ArConnect` issues on me end but here's the [web function](https://github.com/ropats16/arweavekit-examples-test/blob/0183c7af88e2908ac608f716ea6c4dbbed06b01e/src/App.jsx#L41) if needed.